cfme/cloud/instance.py

```diff
@@ -361,6 +361,77 @@ def power_control_from_provider(self, option):
             raise OptionNotAvailable(option + " is not a supported action")
 
 
+@VM.register_for_provider_type("gce")
+class GCEInstance(Instance):
+    # CFME & provider power control options
+    START = "Start"
+    POWER_ON = START  # For compatibility with the infra objects.
+    STOP = "Stop"
+    DELETE = "Delete"
+    TERMINATE = deferred_verpick({
+        version.LOWEST: 'Terminate',
+        '5.6.1': 'Delete',
+    })
+    # CFME-only power control options
+    SOFT_REBOOT = "Soft Reboot"
+    # Provider-only power control options
+    RESTART = "Restart"
+
+    # CFME power states
+    STATE_ON = "on"
+    STATE_OFF = "off"
+    STATE_SUSPENDED = "suspended"
+    STATE_TERMINATED = "terminated"
+    STATE_ARCHIVED = "archived"
+    STATE_UNKNOWN = "unknown"
+
+    def create(self, email=None, first_name=None, last_name=None, availability_zone=None,
+               instance_type=None, cancel=False, cloud_network=None,
```

Comment:
We usually keep the `cancel` attribute as the last one to separate between data that get filled into the form and the final action (cancel / submit).